ci/README.md

```diff
@@ -16,52 +16,52 @@ __Note__: This assumes you are running this the first time, if not, please ensur
 
 1. Clone the `ibc-rs` repo
 
-   
+
 2. Go into the repository folder
-   
+
     `cd ibc-rs`
 
 
 3. Build the relayer container image (this might take a few minutes since it will do a fresh compile and build of all modules)
 
-    `docker-compose -f docker-compose.yml build relayer`
+    `docker-compose -f ci/docker-compose.yml build relayer`
 
 
 4. Run all the containers (two containers, one for each chain and one for the relayer)
 
-   `docker-compose -f docker-compose.yml up -d ibc-0 ibc-1 relayer`
+   `docker-compose -f ci/docker-compose.yml up -d ibc-0 ibc-1 relayer`
 
     If everything is successful you should see a message saying:
     ```shell
     Creating ibc-1 ... done
     Creating ibc-0 ... done
     Creating relayer ... done
     ```
-   
+
     If you want to ensure they are all running properly you can use the command:
-   
+
     `docker-compose ps`
-   
+
```

Comment:
`docker-compose -f ci/docker-compose.yml ps` above and
`docker-compose -f ci/docker-compose.yml down` further down.